test/parallel/test-child-process-cwd.js

```diff
@@ -66,10 +68,25 @@ function testCwd(options, expectPidType, expectCode = 0, expectData) {
     }));
 }
 
+{
+  assert.throws(() => {
+    testCwd({
+      cwd: new URL(`http://${tmpdir.path}`),
+    }, 'number', 0, tmpdir.path);
+  }, /The URL must be of scheme file/);
+
+  assert.throws(() => {
+    testCwd({
+      cwd: new URL(`file://host${tmpdir.path}`),
+    }, 'number', 0, tmpdir.path);
+  }, /File URL host must be "localhost" or empty on linux/);
+}
```

Comment:
```suggestion
  },
  new RegExp(
    `File URL host must be "localhost" or empty on ${process.platform}`));
}
```

Reply:
Because of
```
AssertionError [ERR_ASSERTION]: The input did not match the regular expression /File URL host must be "localhost" or empty on linux/. Input:

'TypeError [ERR_INVALID_FILE_URL_HOST]: File URL host must be "localhost" or empty on darwin'
```

Reply:
```suggestion
  }, /File URL host must be "localhost" or empty/);
}
```

Reply:
done.